python/examples/handler.py

```diff
@@ -45,10 +39,7 @@ def say_ok(request, response, proxy):
     yield print_arg(request, 1)
     yield print_arg(request, 2)
 
-    response.argstreams = [
-        InMemStream(),
-        InMemStream(),
-        InMemStream("world")]
+    response.set_body(InMemStream("world"))
```

Comment:
Yeah, but `set_body` also implies that you're proving the full content. That's different from passing a string to `write_body`.

I mean, `write_body` says you're adding the next chunk of the body. `set_body` says you're providing the full body content (which may be in form of a stream). In fact, if you look at the behavior of `maybe_stream` it closes the stream after building it because you're not going to be writing anything more to the stream.